decidim-meetings/app/controllers/decidim/meetings/admin/meetings_controller.rb

```diff
@@ -57,6 +57,22 @@ def destroy
           end
         end
 
+        def organizers
```

Comment:
Instead of adding a custom action, could you add an `OrganizersController` with an index action? (scoped to a meeting)

Reply:
is this necessary to be scoped to a meeting? if we are returning all the users of the current organization?

Also in `accountability` there are something similar with proposals and there is created as a custom action. 

What is better?

Reply:
Oh, if it's used somewhere else then no need to scope it to a meeting!

Reply:
What about this? I'm OK with not scoping it to a meeting but I'd create an `OrganizersController` anyway.

Reply:
Done! I create a `UsersController` to be reusable to other components